text/0070-x-track-kusamanetwork.md

```diff
@@ -0,0 +1,160 @@
+This would require coordination with whoever controls the X account; they would either need to run the tools themselves or add a third party as an authorized user to
+run the tools to make posts on the account's behalf. This is a bottleneck for decentralization, but as long as the tools are run by the X account manager or by a trusted third party
+it should be fine. I'm open to more decentralized solutions, but those always come at a cost of complexity.
+
+For the tools themselves, we could open a bounty on Kusama for developers/teams to bid on. We could also just ask the community to step up with a Treasury proposal
+to have anyone fund the build. Or, the Fellowship could make the release of these changes contingent on their endorsement of developers/teams to build these tools. Lots of options!
+For the record, me and my team could develop all the necessary tools, but all because I'm proposing these changes doesn't entitle me to funds to build the tools needed
+to implement them. Here's what would be needed:
+
+- a listener tool that would listen for enacted referenda in this track, verify the format of the remark(s), and submit to X's API with authenticating credentials
+- a UI to allow layman users to propose referenda on this track
+
+After everything is complete, we can update the Kusama wiki to include documentation on the X post specifications and include links to the tools/UI.
+
+## Drawbacks
+
+The main drawback to this change is that it requires a lot of off-chain coordination. It's easy enough to include the track on Kusama but it's a totally different
+challenge to make it function as intended. The tools need to be built and the auth tokens need to be managed. It would certainly add an administrative burden to whoever
+manages the X account since they would either need to run the tools themselves or manage auth tokens.
+
+This change also introduces on-going costs to the Treasury since it would need to compensate people to support the tools necessary to facilitate this idea. The ultimate
+question is whether these on-going costs would be worth the ability for KSM holders to make posts on Kusama's X account.
+
+There's also the risk of misconfiguring the track to make referenda too easy to pass, potentially allowing a malicious actor to get content posted on X that violates X's ToS.
+If that happens, we risk getting Kusama banned on X!
+
+This change might also be outside the scope of the Fellowship/openGov. Perhaps the best solution for the X account is to have the Treasury pay for a professional
+agency to manage posts. It wouldn't be decentralized but it would probably be more effective in terms of creating good content.
+
+Finally, this solution is merely pseudo-decentralization since the X account manager would still have ultimate control of the account. It's decentralized insofar as
```

Comment:
The API keys and recovery information could live in an enclave. The enclave would then need to run a light-client listen for remarks.  
Its a bit risky though, since if the enclave messes up then the account access is lost.  

But technically it would be possible to make it fully trustless (% modulo Intel CPU backdors in the enclave 😉).